test.js

```diff
@@ -0,0 +1,15 @@
+import test from 'ava';
+import kapPluginTest from 'kap-plugin-test';
+
+test(async t => {
+	const plugin = kapPluginTest('unicorn.gif');
```

Comment:
It's weird that I can pass in a dummy file while in the plugin a [readStream](https://github.com/wulkano/kap-giphy/blob/master/index.js#L13) is being created for that file. If I change those lines to

```js
const stream = fs.createReadStream('unicorn.gif');

const form = new FormData();
form.append('api_key', context.config.get('apiKey'));
form.append('file', stream);
```

Then it fails off course because that file does not exists. So apparently, it looks like `FormData` is lazy or something. So not sure if I should add a dummy gif just to make sure it exists.